src/components/RouteDisplay.tsx

```diff
@@ -134,47 +134,23 @@ function TransferStep({ action, actions, id, statusData }: TransferStepProps) {
   const renderTransferState = useMemo(() => {
```

Comment:
Technically this could be its own component, but we'll leave it for next PR.

Reply:
yeah but this logic is tight for this component and different between transfer and swap, don't think needs to be it's own component.